tests/test_file_ops.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import os
+from pathlib import Path
+import sys
+sys.path.insert(1, "nesta_ds_utils/")
+import file_ops
+import pytest
+
+
+@pytest.fixture
+def test_output_path():
```

Comment:
There are types available for functions with `yield` ([see here](https://stackoverflow.com/questions/38419654/proper-type-annotation-of-python-functions-with-yield))